client/src/components/index.tsx

```diff
@@ -1,6 +1,5 @@
-import Search from "./Search/Search";
+import Search from "./search/Search";
```

Comment:
This should be `import Search from "./Search/Search";`, not `import Search from "./search/Search";`